### ISSUE TITLE
Next release

### ISSUE THREAD

**Issue by abelino** [tpm2-software/tpm2-tss-engine#290]
Hey @AndreasFuchsTPM. I am working on upstreaming a package for `tpm2-tss-engine` to Buildroot. Do you have a planned/estimated date for the next release cut? I am wanting the following commit: af8b26e7ffe69837197fb841e9a31230ae01c9cc to be part of an official release because without it, I am unable to use the Infineon SLB9673 TPM module.

In the meantime, I am including that commit as a patch as part of the new buildroot package.

Thanks in advance.

**Comment by Hinara** [comment]
Hi,
I'm not related in any way to this project but I would not recommend you to do that as buildroot last version ship OpenSSL3.3.2, and it seems that OpenSSL deprecated the Engine API since OpenSSL 3.0.0
Ref here:
https://github.com/openssl/openssl/blob/openssl-3.0.0/README-ENGINES.md
As such you would be upstreaming a package using a deprecated API which have way more chance to be refused.
If possible try to migrate onto `tpm2-openssl` which is available since version `2024.08.x` of buildroot
https://github.com/buildroot/buildroot/tree/2024.08.x/package/tpm2-openssl
The only trouble might be the command `tpm2tss-genkey` which is only a part of this package

**Comment by hoinmic** [comment]
@abelino Exactly the same with OpenEmbedded, here the commit https://github.com/tpm2-software/tpm2-tss-engine/commit/af8b26e7ffe69837197fb841e9a31230ae01c9cc is applied as a patch too. 
[Openembedded Recipe tpm2-tss-engine with Fix](https://git.yoctoproject.org/meta-security/tree/meta-tpm/recipes-tpm2/tpm2-tss-engine/tpm2-tss-engine_1.2.0.bb?h=master)

@williamcroberts Would a minor release be possible?

**Comment by williamcroberts** [comment]
> [@abelino](https://github.com/abelino) Exactly the same with OpenEmbedded, here the commit [af8b26e](https://github.com/tpm2-software/tpm2-tss-engine/commit/af8b26e7ffe69837197fb841e9a31230ae01c9cc) is applied as a patch too. [Openembedded Recipe tpm2-tss-engine with Fix](https://git.yoctoproject.org/meta-security/tree/meta-tpm/recipes-tpm2/tpm2-tss-engine/tpm2-tss-engine_1.2.0.bb?h=master)
> 
> [@williamcroberts](https://github.com/williamcroberts) Would a minor release be possible?

Yes assuming the CI is passing. I haven't looked at this project in some quite time. We can probably do one more release and then mark it as a dead project since the provider interface supercedes it.

**Comment by hoinmic** [comment]
Sounds like a good plan. thx
